src/ja/user/message/base.py

```diff
@@ -17,6 +20,23 @@ def effective_user(self) -> int:
     def effective_user(self, user: int) -> None:
         self._effective_user = user
 
+    @property
+    def admin_group(self) -> str:
+        return self._admin_group
+
+    @admin_group.setter
+    def admin_group(self, admin_group: str) -> None:
+        self._admin_group = admin_group
+
+    @property
+    def effective_user_is_admin(self) -> bool:
```

Comment:
So, my question is, why duplicate the code here and in the command handler? Can't we just set `user_is_admin` instead of giving the admin group?